### PR TITLE
Update cpu-default-namespace.md

### PR DESCRIPTION
**Pull request by guzj11**
sync with english version

### CODE REVIEW

File: content/zh/docs/tasks/administer-cluster/manage-resources/cpu-default-namespace.md [hunk 1]
```diff
@@ -231,6 +231,23 @@ it can be allowed to run in a namespace that is restricted by a quota.
...
+
+Delete your namespace:
+
+```shell
+kubectl delete namespace default-cpu-example
+```
+-->
+## 清理
+
+删除你的命名空间：
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-cluster/manage-resources/cpu-default-namespace.md:245]: 名字空间，for consistency

File: content/zh/docs/tasks/administer-cluster/manage-resources/cpu-default-namespace.md [hunk 2]
```diff
@@ -263,6 +280,6 @@ it can be allowed to run in a namespace that is restricted by a quota.
 
 * [为容器和 Pod 分配内存资源](/zh/docs/tasks/configure-pod-container/assign-memory-resource/)
 * [为容器和 Pod 分配 CPU 资源](/zh/docs/tasks/configure-pod-container/assign-cpu-resource/)
-* [为 Pod 配置 Service 数量](/zh/docs/tasks/configure-pod-container/quality-service-pod/)
+* [为 Pod 配置服务质量](/zh/docs/tasks/configure-pod-container/quality-service-pod/)
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-cluster/manage-resources/cpu-default-namespace.md:283]: 👍

File: content/zh/docs/tasks/administer-cluster/manage-resources/cpu-default-namespace.md (hunk 1 above)
Comment (tengqm) [reply content/zh/docs/tasks/administer-cluster/manage-resources/cpu-default-namespace.md:245]: This may need a whole page search/replace.

Comment (guzj11) [reply content/zh/docs/tasks/administer-cluster/manage-resources/cpu-default-namespace.md:245]: in other part of this file, is 命名空间 other than 名字空间

Comment (tengqm) [reply content/zh/docs/tasks/administer-cluster/manage-resources/cpu-default-namespace.md:245]: > This may need a whole page search/replace.

Comment (guzj11) [reply content/zh/docs/tasks/administer-cluster/manage-resources/cpu-default-namespace.md:245]: i think 命名空间 is the common translation of  namespace

Comment (guzj11) [reply content/zh/docs/tasks/administer-cluster/manage-resources/cpu-default-namespace.md:245]: > 名字空间，for consistency

is 命名空间 ok ?

Comment (tengqm) [reply content/zh/docs/tasks/administer-cluster/manage-resources/cpu-default-namespace.md:245]: it is fine.

**Pull request merged**